spyder/plugins/ipythonconsole/widgets/shell.py

```diff
@@ -296,7 +296,7 @@ def check_spyder_kernel(self):
         else:
             self.silent_exec_method(code)
 
-    def set_cwd(self, dirname):
+    def set_cwd(self, dirname, update_in_spyder=False):
```

Comment:
Maybe some docstring for this new `update_in_spyder` could be useful? Maybe chaging its name to `emit_cwd_change` could make it more clear?

Reply:
Done.